data/gc/monster_gc.lua

```diff
@@ -0,0 +1,15 @@
+function monster_gc_memory()
```

Comment:
I suggest making loot a table in Lua, and not a class in cpp accessible
through metatable. It would be a lot easier to prevent leak that way.

On Tue, Apr 28, 2015, 16:17 Damian Jarek <notifications@github.com> wrote:

> In data/gc/monster_gc.lua
> <https://github.com/otland/forgottenserver/pull/1296#discussion_r29278261>
> :
>
> > @@ -0,0 +1,15 @@
> > +function monster_gc_memory()
>
> This is useless. By storing Loot and MonsterSpell objects here you prevent
> the garbage collector from cleaning it up until this function is called.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otland/forgottenserver/pull/1296/files#r29278261>.
>